src/Makefile.am

```diff
@@ -512,7 +511,6 @@ libqueso_include_HEADERS += contrib/getpot/getpot.h
 
 nodist_libqueso_include_HEADERS =
 nodist_libqueso_include_HEADERS += $(top_builddir)/config_queso.h
-nodist_libqueso_include_HEADERS += $(top_builddir)/src/core/inc/queso.h
```

Comment:
Oh no, I misunderstood.  Of course we don't ship `config_queso.h` in our tarballs because it's generated by the build system when the user does `./configure && make install`.

As for `queso.h`, I can verify it *is* in the latest (v0.56.0) tarball, so it's not clear to me why the `nodist_` prefix had no effect.